Quantum-Control-Applications/Superconducting/Multi Flux Tunable transmon/Use Case 1 - Two qubit gate optimization with cryoscope/readme.md

```diff
@@ -0,0 +1,188 @@
+# Two qubit chevron improved with cryoscope
+
+*Demonstrated on the experiment of ??.*
+
+*Important note: the code in this folder is the exact code that was used for running this measurement*
+*and it is tailored for a very specific setup and software environment. Thus, the code is here only for insipiration.*
+
+The goal of this use-case is threefold:
+1. to implement Cryoscope [[1](#1), [2](#2)] to measure the step response of the flux line
+2. to design suitable IIR and FIR filters to pre-distort flux pulses

```

Comment:
I think it should say 'filter to distorted flux pulses' using pre-distort sounds like something was done to them but they are just by themselves distorted

Reply:
Actually this is the word used in the cryoscope paper, they talk about predistortion corrections

Reply:
But I'll rephrase it to make it clearer :)